src/main/java/com/hivemq/persistence/clientsession/ClientSession.java

```diff
@@ -69,4 +70,9 @@ public ClientSessionWill getWillPublish() {
     public void setWillPublish(final ClientSessionWill willPublish) {
         this.willPublish = willPublish;
     }
+
+    @Override
+    public int getEstimatedSize() {
+        return 0;
```

Comment:
is the size negligible here? there's still the will contained here, which can be relatively large, right? not sure if this matters

Reply:
its implemented in georgs session branch, sry for that confusing thing....